doc/tutorial/single_cell_recipe.rst

```diff
@@ -12,86 +12,34 @@ and :class:`arbor.simulation` instead of a :class:`arbor.single_cell_model`.
    **Concepts covered in this example:**
 
    1. Building a :class:`arbor.recipe`.
-   2. Using the recipe, context and domain decomposition to create a :class:`arbor.simulation`
+   2. Using the recipe, default context and domain decomposition to create an :class:`arbor.simulation`
    3. Running the simulation and visualizing the results.
 
 The cell
 --------
 
-We can immediately paste the cell description code from the
-:ref:`previous example <tutorialsinglecell-cell>` where it is explained in detail.
+Let's copy the cell description from the :ref:`previous example <tutorialsinglecell-cell>`,
+where construction of the cell is explained in detail.
 
-.. code-block:: python
-
-    import arbor
-
-    # (1) Create a morphology with a single (cylindrical) segment of length=diameter=6 μm
-    tree = arbor.segment_tree()
-    tree.append(arbor.mnpos, arbor.mpoint(-3, 0, 0, 3), arbor.mpoint(3, 0, 0, 3), tag=1)
-
-    # (2) Define the soma and its midpoint
-    labels = arbor.label_dict({'soma':   '(tag 1)',
-                              'midpoint': '(location 0 0.5)'})
-
-    # (3) Create cell and set properties
-    decor = arbor.decor()
-    decor.set_property(Vm=-40)
-    decor.paint('"soma"', 'hh')
-    decor.place('"midpoint"', arbor.iclamp( 10, 2, 0.8), 'iclamp')
-    decor.place('"midpoint"', arbor.spike_detector(-10), 'detector')
-    cell = arbor.cable_cell(tree, labels, decor)
+.. literalinclude:: ../../python/example/single_cell_recipe.py
+   :language: python
+   :lines: 4,8-23
 
 The recipe
 ----------
 
-The :class:`arbor.single_cell_model` of the previous example created a :class:`arbor.recipe` under
-the hood, and abstracted away the details so we were unaware of its existence.
+The :class:`arbor.single_cell_model` of the previous example creates a :class:`arbor.recipe` under
+the hood, and abstracts away a few details that you may want control over in more complex simulations.
+Let's go into which we need to go into so we were unaware of its existence.
```

Comment:
I don't understand this sentence.

Reply:
Me neither :P I hope the rewrite is better ;)